src/hotspot/share/classfile/vmIntrinsics.cpp

```diff
@@ -0,0 +1,862 @@
+/*
+ * Copyright (c) 1997, 2020, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ *
+ */
+
+#include "precompiled.hpp"
+#include "jvm.h"
+#include "classfile/symbolTable.hpp"
+#include "classfile/vmIntrinsics.hpp"
+#include "classfile/vmSymbols.hpp"
+#include "compiler/compilerDirectives.hpp"
+#include "memory/allocation.inline.hpp"
+#include "memory/oopFactory.hpp"
```

Comment:
Can you make sure it needs all of these header files?  I don't see a direct reference to SymbolTable:: or OopFactory.

Reply:
Done. Removed a bunch of them!